math256/build.sh

```diff
@@ -1,10 +1,17 @@
 #!/usr/bin/env bash
 set -o errexit -o nounset -o pipefail
 
-# This script builds all subprojects and puts all created Wasm modules in one dir
+# This script validates and builds all subprojects and puts all created Wasm modules in one dir
+
 cargo update
+
+cargo clippy --all -- -D warnings 
+cargo fmt --all 
+# tests are fast, so run them here
+cargo test
```

Comment:
this is going to fail since the modules don't exist.
1. marine build --release
2. cargo +nightly test --release

Reply:
will do clean git run, thanks